src/main/java/com/example/studentmanagmentrest/key/model/SecretKey.java

```diff
@@ -0,0 +1,25 @@
+package com.example.studentmanagmentrest.key.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+public class SecretKey {
+
```

Comment:
Suggest using sequences for postgres DB
/////////
    @Id
    @SequenceGenerator(
            name = "secretKey",
            sequenceName = "secretKey_sequence",
            allocationSize = 1
    )
    @GeneratedValue(
            strategy = GenerationType.SEQUENCE,
            generator = "secretKey_sequence"
    )
    @Column(nullable = false,updatable = false)
    Long id;